include/solvers/navier_stokes_scratch_data.h

```diff
@@ -839,14 +921,167 @@ class NavierStokesScratchData : public PhysicsScratchDataBase
             fluid_particle_relative_velocity_at_particle_location[particle_i]
               .norm() *
             particle_properties[DEM::CFDDEMProperties::PropertiesIndex::dp] /
-            (kinematic_viscosity + DBL_MIN);
+            (kinematic_viscosity_at_particle_location[particle_i] + DBL_MIN);
         particle_i++;
       }
 
     average_fluid_particle_relative_velocity =
       average_fluid_particle_relative_velocity / particle_i;
   }
 
+  /** @brief Interpolates the filtered phase fraction value at the location of the particles.
+   * The latter values are used in calculating the density and viscosity of the
+   * fluid at the particles' locations when VOF is used.
+   *
+   * @param q_particles Quadrature type object that contains the location of the particles
+   * relative to the cell's frame of reference.
+   *
+   * @param phase_cell The active cell associated with the vof DoFHandler
+   *
+   * @param current_filtered_solution The present value of the solution for the filtered phase
+   * fraction
```

Comment:
Well that for the VOF right? I would be more verbose in the wording here because the phase fraction to me could be related to the void fraction. I would say the filtered VOF solution.